Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "iota-crypto"
-version = "0.23.0"
+version = "0.23.1"
```

Comment:
We do not update version manually. See [.changes](https://github.com/iotaledger/crypto.rs/blob/dev/.changes/readme.md).

Reply:
right, thanks